README.md

```diff
@@ -1,8 +1,9 @@
 # infusion-icons #
-SVG icons for use in Infusion
+SVG icons which can be used directly or compiled into an icon font.
+The icons have been created based on the [Infusion Icons Visual Style Guide](https://wiki.fluidproject.org/display/fluid/Infusion+Icons+Visual+Style+Guide).
 
 ## Documentation ##
```

Comment:
Would it be useful to include some documentation on how the repository / icons can be used? In particular, is there a recommended way? (i.e. copy the SVGs? use github? other?). What if you wanted to use just a subset of the icons?

Reply:
I think we're still working through those issues unfortunately. http://fluid.2324889.n4.nabble.com/Icon-font-building-tt10242.html